opentracing-mock/src/test/java/io/opentracing/mock/MockSpanTest.java

```diff
@@ -26,55 +26,59 @@
     @Test
     public void testSetOperationNameAfterFinish() {
         MockTracer tracer = new MockTracer();
-        Span span = tracer.buildSpan("foo").start();
+        Span span = tracer.buildSpan("foo").startManual();
         span.finish();
 
         try {
             span.setOperationName("bar");
             Assert.fail();
         } catch (RuntimeException ex) {
+            // Fall through
         }
         Assert.assertEquals(1, tracer.finishedSpans().get(0).generatedErrors().size());
     }
 
     @Test
     public void testSetTagAfterFinish() {
         MockTracer tracer = new MockTracer();
-        Span span = tracer.buildSpan("foo").start();
+        Span span = tracer.buildSpan("foo").startManual();
```

Comment:
On all the `start()` -> `startManual()` changes: Shouldn't we leave at least some regression tests for the deprecated behaviour in place? Possibly in a new test class.

Reply:
Sure. :+1: